.circleci/config.yml

```diff
@@ -73,12 +81,12 @@ jobs:
           name: Run tests
           command: npm test
 
-  npm:
+  publish_package:
     <<: *defaults
     steps:
     - run:
```

Comment:
Should these `steps` be indented one level? Looks like that's how they're formatted above in the `build_test` job.